packages/webapp/__tests__/SquadTokenPage.tsx

```diff
@@ -175,6 +175,7 @@ describe('squad details', () => {
     const owner = generateTestOwner();
     const members = [generateTestMember('u1'), generateTestMember('u2')];
     owner.source.members.edges.push({ node: members[0] }, { node: members[1] });
+    owner.source.membersCount = 3;
```

Comment:
Should this not come from the graphQL query?

Reply:
Yes, this will be part of the GraphQL query. We will pass this `owner` to the mock on
```
renderComponent([createInvitationMock(defaultToken, owner)]);
```

Reply:
Ah it's mocked I see, was thinking the actual data would already return it.